build/shards.zsh

```diff
@@ -0,0 +1,25 @@
+
+function shards.build {
+	write "${target}: ${src[@]} ${depends[$target]}"
+	write "\t@echo '${fg_bold[magenta]}  [SHARD]    ${fg_bold[white]}$target${reset_color}'"
+	write "\t${Q}shards build"
```

Comment:
Is not related to the target. You could have a `make foo` that doesn’t build foo.